Userland/Libraries/LibWeb/Painting/PaintingCommandExecutorCPU.h

```diff
@@ -48,6 +48,9 @@ class PaintingCommandExecutorCPU : public PaintingCommandExecutor {
 
     bool would_be_fully_clipped_by_painter(Gfx::IntRect) const override;
 
+    bool wants_prepare_glyphs_texture() const override { return false; }
```

Comment:
maybe `needs_prepare_glyphs_texture`, `wants` sounds a bit weird to me

Reply:
yeah, `needs` looks more appropriate. I changed that.